packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -108,10 +108,10 @@ export function addSpanNetworkEvents(
  * sort resources by startTime
  * @param filteredResources
  */
-export function sortResources(filteredResources: PerformanceResourceTiming[]) {
-  return filteredResources.slice().sort((a, b) => {
-    const valueA = a[PTN.FETCH_START];
-    const valueB = b[PTN.FETCH_START];
+export function sortResources(filteredResources: PerformanceResourceTiming[]): PerformanceResourceTiming[] {
+  return filteredResources.slice().sort((x, y) => {
+    const valueA = x[PTN.FETCH_START];
```

Comment:
was change from a,b to x,y because of lint ??

Reply:
I don't see the need for the change

Reply:
variable `a` shadows the `a` declared in line 32.
Actually, I think you are right and probably it is better to rename the one in line 32 to something like `anchorElement`.

Reply:
I renamed it to `urlNormalizingAnchor`.